### PR TITLE
Update data feed instructions

### PR DESCRIPTION
**Pull request by alexandernacho**
AWS changed the way S3 buckets have to be set up. For that reason, our docs was not up-to-date anymore.

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -286,10 +286,10 @@ If you do not already have a cloud provider, we suggest working with Amazon Web
 1. Open [https://s3.console.aws.amazon.com/s3/buckets/](https://s3.console.aws.amazon.com/s3/buckets/)
 2. Create a new bucket in the region of your choice. We suggest calling the bucket **kaiko-delivery-nameofyourcompany**
 3. We suggest you use the us-east-1 region unless you already are using a different region as your main presence
-4. Leave the properties on the default setting
-5. On the **Set permissions** tab press **Add account** and use the following id: <code>4d6be087cf0b9ee7af2f8d8c51469c81bc711e68ee9c6be386aee2322abc8175</code>
-6. Check all the checkboxes for permissions and press **Save**
-7. Ensure that the permissions dialog resembles the following: ![AWS Dialog](/images/S3_Management_Console_large.png)
```
Comment (Fonsan) [inline source/index.html.md]: Why no screenshot?

File: source/index.html.md [hunk 2]
```diff
@@ -286,10 +286,10 @@ If you do not already have a cloud provider, we suggest working with Amazon Web
...
+4. Leave the properties on the default setting, finish the review and click on 'Create bucket'
+5. After creating the bucket, click on it and navigate to the 'Permissions' tab.
```
Comment (jmouvill) [inline source/index.html.md]: May be add another intermediary step for the `Access Control List` sub tab ?

**Comment by jmouvill** [comment]
The updated screenshot is committed in another branch/PR : https://github.com/challengerdeep/docs.kaiko.com/pull/43 
Is this intended ?

**Comment by alexandernacho** [comment]
@jmouvill It was not. Fixed this

**Pull request merged**